model/shop.php

```diff
@@ -139,16 +139,16 @@ function __construct(Uzivatel $u, $nastaveni = null)
         );
         $fronta = &$this->tricka[];
         // hack pro výběr správného automaticky objednaného trička
-        if($smiCervene)                 $barva = 'červené';
-        elseif($smiModre)               $barva = 'modré';
-        else                            $barva = '.*';
-        if($this->u->pohlavi() == 'f')  $typTricka = 'tílko.*dámské S';
-        else                            $typTricka = 'tričko.*pánské L';
-        $r['auto'] = (
-          $r['nabizet'] &&
-          preg_match("@$barva@i", $r['nazev']) &&
-          preg_match("@$typTricka@i", $r['nazev'])
-        );
+        // if($smiCervene)                 $barva = 'červené';
```

Comment:
Co to dát na nějaký přepínač? Podmínit to konstantou (které používáme jako options), třeba `AUTOMATICKY_VYBER_TRICKA` a defaultně false.

Se zakomentovaným kódem nemám dobré zkušenosti. Minimálně to brnká o nervy jako něco, co by mělo  být mrtvé, ale není vůle to za mrtvé prohlásit.